### ISSUE TITLE
GO:0000030 | mannosyltransferase activity , PTN002003922

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5293]
* **PTHR ID & PTN node:**

GO:0000030 | mannosyltransferase activity | IBA with S000000557 , PTN002003922 | contributes_to | GO_REF:0000033

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPCC1919.02 pbn1/ PIG-X
https://www.yeastgenome.org/locus/S000000557 Pbn1

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

Annotation

@edwong57 

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
sorry the details:

UniProt Human PIG-M concurs
Mannosyltransferase involved in glycosylphosphatidylinositol-anchor biosynthesis. Transfers the first alpha-1,4-mannose to GlcN-acyl-PI during GPI precursor assembly.
PMID: 11226175

PIG-M (gpi14) and PIG-X (Pbn1)

are the 2 members of
GO:1990529 glycosylphosphatidylinositol-mannosyltransferase I complex

PIG-X appears to be the regulatory protein
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR013233/

see https://github.com/geneontology/go-ontology/issues/28214

**Comment by edwong57** [comment]
What do you think is an appropriate replacement for 'mannosyltransferase activity' for PBN1? GO:0005198 structural molecule activity?

**Comment by ValWood** [comment]
I was thinking 
GO:0030234   enzyme regulator activity
Molecular Function
Definition ([GO:0030234 GONUTS page](http://gowiki.tamu.edu/wiki/index.php/Category:GO:0030234))
Binds to and modulates the activity of an enzyme.

but I'm not sure if this is correct if it has only been shown to affect stability? it seems more like a "scaffold" and we don't have such a term (it is conflated with adaptor...I have issue with this https://docs.google.com/document/d/1Qakmsk19Sr4H_2cDpzMU0bi8duD7Rbye5aJ_Rne-7ZE/edit?usp=sharing
because not all scaffold proteins are adaptors

GO:0060090 molecular adaptor activity
Has related synonyms
protein complex scaffold activity
And
protein-containing complex scaffold activity

but this could be an example of a scaffold that isn't an adaptor.

@pgaudet what do you think?

**Comment by marcfeuermann** [comment]
I would indeed rather use GO:0060090 molecular adaptor activity.
At the PAINT level, annotation has been removed.
Feel free to further discuss or close this ticket.

**Comment by ValWood** [comment]
The problem with adaptor is that it implies a connection of 1 to B, and that does not seem to be the case here.
This is an ontology issue though...

**Comment by ValWood** [comment]
Although quite a few of these so called regulatory subunits are actually membrane adaptors for the catalytic subunt... I will see if I can find info about this for this one...

**Comment by ValWood** [comment]
still not clear from the 2 most recent papers I could find
https://pubmed.ncbi.nlm.nih.gov/35420475/ Leishmania donovani Ortholog 2022
https://pubmed.ncbi.nlm.nih.gov/34875117/ T. brucei 2022

**Comment by edwong57** [comment]
Yeah, I'm kind of getting hung up on the definition of 'molecular adaptor activity' stating  "The binding activity of a molecule that brings together two or more molecules..." In these cases, that doesn't seem to be happening.

**Comment by ValWood** [comment]
Perhaps enzyme regulator is best?

This does not even seem to localize  gpi14 to the membrane because GPI14 is a multimenbrane-spanning protein.
We know it is  absolutely required for the activity, but we don't know the precise  mechanism 
@pgaudet  is it OK to use enzyme regulator activity in these cases? It doesn't have catalytic activity, and it doesn't appear to be an adaptor - there isn't really any information on how it affects/contributes to  the activity of gpi14
